doc/api/timers.md

```diff
@@ -162,6 +162,50 @@ added: v0.0.1
 
 Cancels a `Timeout` object created by [`setTimeout()`][].
 
+## Manual Timers
+
+It is possible to make any object a timer. Reusing an existing object as a timer
```

Comment:
I would say something like, `It is possible, but not recommended, to make any object into a timer.`